srcCpp/CustomType.h

```diff
@@ -0,0 +1,40 @@
+#ifndef __CUSTOMTYPE_H__
+#define __CUSTOMTYPE_H__
+/*
+ * (c) 2005-2018  Copyright, Real-Time Innovations, Inc. All rights reserved.
+ * Subject to Eclipse Public License v1.0; see LICENSE.md for details.
+ */
+#define Q(x) #x
+#define QUOTE(x) Q(x) // Used to set quotation marks
+
+#include QUOTE(RTI_CUSTOM_TYPE_FILE_NAME_SUPPORT)
+
+#include "perftest.h"
+#include "perftest_cpp.h"
+#include <string>
+
+/*
+ * It is the header files that contain the definition of the requires and
+ * optional functions used for the handling of the custom type. These functions
+ * will be used only if the parameter "--customType <type>" is applied in the
+ * build. You can find more information in the point "Using Custom Types" of the
+ * section "Use-Cases And Examples" in the documentation.
+ */
```

Comment:
This file contains the definition of the API require to work with Custom Type. 
These functions will be used only if the parameter "--customType <type>" is applied in the build is used.
You can find more information in the section "..." under ...